blog/views.py

```diff
@@ -109,10 +129,14 @@ def viewpost(request, post_id):
         content = commonmark.commonmark(post.content)
     else:
         content = post.content
+        if post.image != None:
+
+            image = post.image.__str__()
 
     return render(request, "blog/viewpost.html",
                   {"post": post, "edit": author == post.author,
-                   "content": content})
+                   "content": content,
+                   "image":  '../../../media/' + image}) # temporally hardcoding the path
```

Comment:
use `polarbear.settings.MEDIA_ROOT`

Reply:
This seems not working, Django cannot find the URL using polarbear.settings.MEDIA_ROOT

Reply:
what is the value of `polarbear.settings.MEDIA_ROOT`?

Reply:
/Users/username/Desktop/CMPUT404-project/media/

Reply:
that should work, shouldn't it?

Reply:
I think the root path should be the root directory of our project instead of the root of OS

Reply:
I'm fine with this for now